src/app/api/imageInput/route.ts

```diff
@@ -104,23 +104,23 @@ export async function POST(request: Request, response: NextApiResponse) {
       },
       onCompletion: async (fullResponse: string) => {
         console.log("onCompletion", fullResponse);
-        const latestReponse = {
-          id: id,
-          role: "assistant" as const,
-          content: fullResponse,
-          createdAt: new Date(),
-          audio: "",
-        };
-        const db = await saveToDB({
-          _chat: _message,
-          chatId: chatId,
-          orgSlug: orgSlug as string,
-          latestResponse: latestReponse,
-          userId: userId,
-          orgId: orgId,
-          urlArray: urlArray,
-        });
-        console.log("dbResponce", db);
+        // const latestReponse = {
```

Comment:
why is this commented ?
we decided to remove that extra call from frontend side not from here

Reply:
Resolved